src/content/docs/en/pages/guides/open-ticket/open-ticket.mdx

```diff
@@ -28,66 +26,51 @@ To create a ticket, you first need to access **Support Center**. You have two op
 ### Through Azion Console
 
 1. Access [Azion Console](https://console.azion.com/).
-2. On the header, click on the Help Center **(?)** icon > select **Contact Support**.
+2. On the header, click **Help** > **Contact Support**.
 
 ---
 
 ## Opening a Ticket
 
-After you log in to your account, you're redirected to the **Submit a ticket** page. Click **+ New support ticket** on the right-side of the search field and fill in the field as described next:
+After you log in to your Azion account, you'll be redirected to the **Submit a ticket** page. Click **+ New support ticket** to the right of the search field and fill in the field as follows:
 
 | Field | Description |
-|-------|-----------|
-| Requester* | Automatically filled with your Azion login email. |
-| Product* | List of Azion products. You can choose one product per ticket. |
+| --- | --- |
+| Requester* | This field is automatically filled in with your Azion login email. |
+| Product* | Select the Azion product from the list. You can choose one product per ticket. |
 | Subject* | Add a title that describes your problem to identify your ticket. |
-| Severity* | General Guidance<br /> System impaired<br /> Production system impaired<br /> Production system down<br /> Business-critical system down<br /><br />See the description of each of the [severity levels](#severity-levels) next. |
-| Type* | Reason for opening the ticket:<br /><br />- Incident: to report an incident. <br />- Question: to clarify your doubts. <br />- Request: to make a specific request; for example, request a new integration or feature. |
-| Description* | Open field to describe your needs. Add all information about your problem, question, or request, to help agents understand how to help you. |
+| Type* | Select the reason for opening the ticket:<br /><br /><ul><li>**Incident:** to report an incident.</li><li>**Question:** to clarify your question.</li><li>**Request:** to make a specific request; for example, request a new integration or feature.</li></ul> |
+| Description* | Detail your needs in this field. Include as much information as possible about your incident, question, or request to assist agents in understanding how best to support you. |
 | + Attach a file<br /> *optional* | Add a file to help describe your ticket. It's recommended to use JPEG, PNG, or PDF file extensions. |
-| Phone number | Add the phone number of the person responsible for discussing the problem you're reporting. |
-
-Select the **Allow contact by phone** check box if you want to be contacted through your phone number, in case Azion agents find it necessary.
-
-> Fields marked with an asterisk are required.
-
-After filling in the form:
+| Phone number | Enter the phone number of the individual responsible for discussing the reported problem. |
+| Allow contact by phone | Enable this check box if you wish to be contacted via the provided phone number, should Azion agents deem it necessary for the case. |
```

Comment:
```suggestion
| Allow contact by phone | Enable this check box if you wish to be contacted via the provided phone number, in case Azion agents find it necessary for the case. |
```